.travis.yml

```diff
@@ -1,20 +1,32 @@
 dist: trusty
 language: go
-os:
-  - windows
-  - linux
-  - osx
-
-go:
-  - 1.12.x
-
 go_import_path: github.com/prometheus/tsdb
+matrix:
+  include:
+  - os: linux
+    dist: trusty
+    sudo: required
+    go: 1.12.x
+    install:
+      - make deps
+    script:
+      - sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install fuse
+      - sudo modprobe fuse
+      - sudo chmod 666 /dev/fuse
+      - sudo chown root:$USER /etc/fuse.conf
+      - sudo addgroup $USER fuse
+      - sudo chmod a+rw /etc/fuse.conf
```

Comment:
why do we need these?
```
- sudo chmod 666 /dev/fuse
- sudo chown root:$USER /etc/fuse.conf
- sudo chmod a+rw /etc/fuse.conf
```

I think just `- sudo addgroup $USER fuse` should be enough , no?

Reply:
We need to write user_allow_other to fuse.conf to allow other users which is not root to execute fuse

Reply:
are you sure that `sudo echo user_allow_other >> /etc/fuse.conf` doesn't work without `sudo chmod a+rw /etc/fuse.conf` ?